aigerbv/common.py

```diff
@@ -401,11 +419,21 @@ def abs_gate(wordlen, input, output):
         >> bitwise_xor(wordlen, output, tmp2, output)
 
 
-def lookup(wordlen, mapping, input, output):
+def lookup(inlen, outlen, mapping, input, output, *,
+           in_signed=True, out_signed=True):
```

Comment:
Maybe add an example as a comment how to use this function. The function is a bit too complicated to figure out what the star does.

Reply:
`*` means keyword only argument in python 3. It's useful for adding features later. I particularly like it for cases where I think not using keywords makes things confusing.

Reply:
That said, I agree, I should document this function better.